docs/docs/user/migration-guide/v6-v7.md

```diff
@@ -60,3 +60,40 @@ As a result of the restructure, some imports need to be adjusted:
 -components/__init__.py
 +kpops/components/custom/__init__.py
 ```
+
+## [Call destroy from inside of reset or clean](https://github.com/bakdata/kpops/pull/501)
+
+Before v7, the KPOps CLI executed `destroy` before running `reset/clean` to ensure the component was destroyed.
+
+This logic has changed. The `destroy` method is now called within the `PipelineComponent`'s `reset/clean`.
+
+During migrating to v7, you should check your custom components and see if they override the `reset/clean` methods. If so, you need to call the supermethod `reset/clean` to trigger the `destroy` inside the parent class. Alternatively, if you are implementing the `PipelineComponent` class, you need to call the `destroy` method at the beginning of the method.
+
+#### components.py
+
+For example, when creating a custom `StreamsApp` or `ProducerApp` (or any other custom component), you **must** call the supermethod `rest/clean` to execute the `destroy` in the parent class. **Otherwise, the logic of destroy will not be executed!**
```

Comment:
```suggestion
For example, when creating a custom `StreamsApp` or `ProducerApp` (or any other custom component), you **must** call the supermethod `reset`/`clean` to execute the `destroy` in the parent class. **Otherwise, the logic of destroy will not be executed!**
```